HornsAndHooves-slackiq.gemspec

```diff
@@ -1,8 +1,8 @@
 # -*- encoding: utf-8 -*-
 
-lib = File.expand_path("../lib", __FILE__)
+lib = File.expand_path("lib", __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require "slackiq/version"
```

Comment:
Typically gems store the version in their own file under <gemname>/version.rb. They usually only contain a version constant and that is fine.